Source/JavaScriptCore/llint/LLIntThunks.cpp

```diff
@@ -672,7 +672,7 @@ MacroAssemblerCodeRef<NativeToJITGatePtrTag> tagGateThunk(void* pointer)
     jit.tagPtr(GPRInfo::regT3, ARM64Registers::lr);
     jit.storePtr(ARM64Registers::lr, CCallHelpers::Address(GPRInfo::callFrameRegister, 8));
     jit.move(CCallHelpers::TrustedImmPtr(pointer), GPRInfo::regT3);
-    jit.farJump(GPRInfo::regT3, OperationPtrTag);
```

Comment:
I'm unsure of whether this change, as well as the one in untagGateThunk, are correct. W.r.t. this one: Is the idea that this function is called once to tag all of the thunks? If so, for what reason would it farJump out somewhere else rather than just returning back?